src/app/store/tabsSlice.ts

```diff
@@ -333,7 +331,6 @@ const tabsSlice = createSlice({
   extraReducers(builder) {
     builder.addCase(
       queryTableland.fulfilled,
-      // @ts-ignore
       (state, { payload: { columns, rows, query, tabId, error } }) => {
```

Comment:
@joewagner i ended up adding linting....there was quite a lot to resolve! i've fixed everything but can't seem to figure out how to fix this last error. i'm guessing it's either the `undefined` part of the `type 'TablelandQueryResult | undefined'` that's expected, or it might be something with the fact `TablelandQueryResult` extends another type.

it complains with the following, in case you have any idea on how to resolve:
```md
ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts
./src/app/store/tabsSlice.ts 334:27-34
[tsl] ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts(334,28)
      TS2339: Property 'columns' does not exist on type 'TablelandQueryResult | undefined'.

ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts
./src/app/store/tabsSlice.ts 334:36-40
[tsl] ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts(334,37)
      TS2339: Property 'rows' does not exist on type 'TablelandQueryResult | undefined'.

ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts
./src/app/store/tabsSlice.ts 334:42-47
[tsl] ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts(334,43)
      TS2339: Property 'query' does not exist on type 'TablelandQueryResult | undefined'.

ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts
./src/app/store/tabsSlice.ts 334:49-54
[tsl] ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts(334,50)
      TS2339: Property 'tabId' does not exist on type 'TablelandQueryResult | undefined'.

ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts
./src/app/store/tabsSlice.ts 334:56-61
[tsl] ERROR in /Users/dtb/tbl/console/src/app/store/tabsSlice.ts(334,57)
      TS2339: Property 'error' does not exist on type 'TablelandQueryResult | undefined'.
```